.gitignore

```diff
@@ -165,3 +165,5 @@ cython_debug/
 # The secret-containing files
 canary_config.py
 slack_reporter_config.py
+
+viam-server
```

Comment:
Nitpicking: this file (and frankly all files) should end in a newline, to avoid spurious diffs next time someone else edits it. Instructions to do this on many text editors are [at the bottom of this link](https://thoughtbot.com/blog/no-newline-at-end-of-file), and I'm happy to help you set this up if you want.

Reply:
thanks I just turned this on in vscode!